other/docker/windows/build_toxcore.sh

```diff
@@ -45,6 +45,8 @@ build() {
   fi
 
   cp -a "$TOXCORE_DIR" /tmp/toxcore
```

Comment:
@nurupo The issue occurs here. When you're mounting the c-toxcore directory from the host to the docker container /toxcore path, it will copy all of the files from it(including, CMakeCache.txt and the CMakeFiles directory if those exist) to /tmp/toxcore in order to compile toxcore in that path. If you've compiled toxcore on linux and then attempt to do a cross-compilation you will receive the error message shown in the pr description. I didn't want to cause problems with the build system so I chose to just delete the cache files after copying instead. Let me know if there is a better way to do this.